modules/oadp-backing-up-pvs-csi.adoc

```diff
@@ -7,7 +7,7 @@
 = Backing up persistent volumes with CSI snapshots
 :context: backing-up-applications
 
-You back up persistent volumes with Container Storage Interface (CSI) snapshots by editing the `VolumeSnapshotClass` custom resource (CR) of the cloud storage before you create the `Backup` CR.
+You back up persistent volumes with Container Storage Interface (CSI) snapshots by editing the `VolumeSnapshotClass` (VSC) custom resource (CR) of the cloud storage before you create the `Backup` CR.
```

Comment:
```suggestion
You back up persistent volumes with Container Storage Interface (CSI) snapshots by editing the `VolumeSnapshotClass` (VSClass) custom resource (CR) of the cloud storage before you create the `Backup` CR.
```

Reply:
I removed the VSC completely. I shouldn't have added it. It's not used in the documentation. My mistake.